specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -6773,6 +6807,20 @@
       },
       "description": "The instance view of a dedicated host."
     },
+    "DedicatedHostInstanceViewWithName": {
+      "properties": {
+        "name": {
```

Comment:
>"name" [](start = 8, length = 6)

why is this just the name and not the full resource ID?

Reply:
Dedicated host always belong to the same subscriptionId and the resourceGroup as its parent dedicated host group, and this "name" property is only shown as a part of the response to dedicated host group instance view.